### PR TITLE
[3.8] Fix typos in Misc/NEWS.d

### PR DESCRIPTION
**Pull request by toonarmycaptain**
With reference to corrections to [news in 3.9.0a2.rst](https://github.com/python/cpython/pull/17665) backporting some typo corrections to 3.8.0/3.8.1.

### CODE REVIEW

**Review by aeros — CHANGES_REQUESTED** [review]
Thanks for the PR @toonarmycaptain.

Could you prefix the PR title with `[3.8]`? This would look like: `[3.8] Backport typo corrections`. This format is required for any PR opened against a maintenance branch, otherwise it won't pass @bedevere-bot's checks. 

I would also consider adjusting to title to something more informative of the PR, such as `[3.8] Fix typos in Misc/NEWS.d`. "typo corrections" doesn't explain much about the location of the changes, as that could apply to any part of the documentation, code comments, docstrings, etc. 

I believe it is correct to specifically open the PR against the 3.8 branch, as changes to Misc/NEWS.d are typically only applied to branch's versions rather than all versions. This differs from other areas, where the changes should be applied to master and then backported to previous versions to all relevant locations.

I'm not certain about the typo fix that adjusts the number of spaces between sentences. IIRC, it still renders exactly the same in reST; @merwok or @willingc might be able to clarify on that. 

The other changes LGTM though, thanks!

**Comment by toonarmycaptain** [comment]
Title change done. 
I can remove the space change if you like? It was just inconsistent, so I changed it. I made the PR referenced above against 3.9, and was just trying to find the corrections from that PR and bring them to 3.8 (so that they actually change in the 3.8 docs), as like you'd said I'd expected it to carry back.

**Comment by aeros** [comment]
@toonarmycaptain 

> Title change done.

Thanks.

> I can remove the space change if you like? It was just inconsistent, so I changed it.

You can wait for additional feedback on that, but generally speaking we try to avoid making changes *purely* for the sake of consistency. This is significantly more important for changes to the source code, but it also applies to documentation or news changes. That's why I @ mentioned a couple of core developers that were especially experienced with reST, to check if the number of spaces makes a difference there.

One small unnecessary change may not seem like a big deal, but it can really add up over time. PR review time comes at a premium, especially for a project ran almost entirely by voluntary efforts. It improves our workflow significantly when PRs have a more focused scope and avoid any unneeded changes.

That being said, there's no harm at all in waiting for a second opinion; we have to wait on approval from a core developer before it can be merged.

**Review by aeros — APPROVED** [review]

**Comment by toonarmycaptain** [comment]
> That being said, there's no harm at all in waiting for a second opinion; we have to wait on approval from a core developer before it can be merged.

No problem, no need to both the core devs unnecessarily! I just left it up to you as you were reviewing.  

Looking at https://docs.python.org/3.8/whatsnew/changelog.html#python-3-8-1-final, and [this SO question](https://stackoverflow.com/questions/15721373/how-do-i-ensure-that-whitespace-is-preserved-in-markdown), there's one space in my browser, not three, so it appears as if the extra whitespace is collapsed either when used to generate the HTML or when the HTML is rendered.

**Review by asvetlov — APPROVED** [review]

**Comment by asvetlov** [comment]
Thanks!

**Pull request merged**
